src/store/index.ts

```diff
@@ -18,7 +23,12 @@ export function getVuexStore () {
     state: { ...State },
     getters: { ...Getters },
     mutations: { ...Mutations },
-    actions: { ...Actions }
+    actions: { ...Actions },
+    modules: {
+      dissolution: DissolutionModule,
+      incorporation: IncorporationModule,
+      registration: RegistrationModule
```

Comment:
I think you can see where this is going, muhahaha.

For now, this doesn't seem to hurt anything.

Reply:
It remains to be seen whether it's easier/cleaner to use separate modules to store separate filing data...